lib/bugsnag/loggers/logger.rb

```diff
@@ -0,0 +1,53 @@
+require "logger"
+require "bugsnag"
+require "bugsnag/loggers/log_device"
+
+module Bugsnag::Loggers
+
+  SEVERITIES = [
+    "debug",
+    "info",
+    "warn",
+    "error",
+    "fatal",
+    "unknown"
+  ]
+
+  class Logger < Logger
+
+    def initialize(level=Logger::INFO)
+      @open = true
+      @log_device = Bugsnag::Loggers::LogDevice.new
+      super @log_device, level
```

Comment:
What is the benefit of having the device over `nil` here? The `add()` method could call `Bugsnag.leave_breadcrumb` directly.

Reply:
You're right, there is no advantage.  I'll remove this.  For some reason I assumed the logger base class wouldn't be happy being passed nil, but it's quite happy.